docker/tensorflow-application/video-serving/streaming.py

```diff
@@ -0,0 +1,61 @@
+from flask import Flask, Response, request
+from flask import render_template, send_file
+import threading
+import argparse
+import datetime
+import imutils
+import time
+import cv2
+import requests
+import numpy as np
+from PIL import Image
+from flask import jsonify
+import time
+
+
+outputFrame = None 
+lock = threading.Lock()
+
+app = Flask(__name__, template_folder='templates')
+
+@app.route("/")
+def index():
+	return render_template('index.html')
+
+def stream_to_memory():
+	global outputFrame, lock	
+	list1 = request.json['instances']
+	for l in list1:
+		img=np.asarray(l, dtype=np.uint32)
+		outputFrame = Image.fromarray(img.astype('uint8'))
+
+
+@app.route("/video")
+def video():
+	def produce_frame(): 
+		global outputFrame, lock 
```

Comment:
You aren't using the lock?

Reply:
I missed out on removing this part of the code. My first iteration had locks but I don't think we need it here.